docs/about/activities.md

```diff
@@ -17,10 +17,13 @@ title: Activities
   Sciences) (2019). [PDF](https://arxiv.org/pdf/1909.04032.pdf)
 - **Texterkennungssoftware für historische Drucke.** Wehner,
   Maximilian in KulturBetrieb 25 (2019). [PDF](http://www.kulturbetrieb-magazin.de/fileadmin/user_upload/kulturbetrieb-magazin/magazin/KulturBetrieb-2019-Ausgabe-2-November.pdf)
+You can find further related publications [here](https://www.uni-wuerzburg.de/zpd/zentrum/team/reul-christian/publications/).
```

Comment:
This entry either needs an additional new line between the list or a bullet point. Otherwise the markdown parser just adds it directly behind the text of the previous list entry.